episodes/introduction.md

```diff
@@ -38,7 +41,34 @@ and their past experience with curriculum development and The Carpentries.
 
 :::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
 
-FIXME: Add more text about the reading each week and discussion sessions to set the stage for how trainer training will run.
+
+## About this Training
+
+The goal of this training is to prepare participants to teach
+[The Carpentries Collaborative Lesson Development Training][cldt] (CLDT).
+
+The training will involve weekly discussion calls 
+attended by trainees and facilitators,
+as well as reading tasks and occasional, more practical, homework tasks.
+The reading and other tasks have been selected to 
+provide trainees with insight into the CLDT curriculum,
+some of the literature on which it is based,
+and the experience participants will have when following the lesson development training.
```

Comment:
```suggestion
and the experience participants will have when in the lesson development training.
```
not required, but "following" sounds a bit like they could be doing self-paced to me